activesupport/test/core_ext/hash_ext_test.rb

```diff
@@ -963,8 +963,12 @@ def test_except_with_original_frozen
 
   def test_except_with_mocha_expectation_on_original
     original = { :a => 'x', :b => 'y' }
-    original.expects(:delete).never
-    original.except(:a)
+    mock = Minitest::Mock.new
+    mock.expect(:call, nil, [])
+    original.stub(:delete, mock) do
+      original.except(:a)
+    end
+    assert_equal "expected call() => nil, got []", assert_raises(MockExpectationError) { mock.verify }.message
```

Comment:
This definitely doesn't feel right to me. The mocking above also adds a lot of noise to the test. How about:

```ruby
delete_called = false

original.stub(:delete, -> { delete_called = true }) do
  original.except(:a)
end

assert_not delete_called
```